source/horoscope/css/landing.css

```diff
@@ -141,41 +141,53 @@ input[type="radio"]:checked + label {
     font-size: 1.5em;
 }
 
-nav {
-    position: fixed;
-    bottom: 20px;
-    right: 20px;
-    display: flex;
-    flex-direction: column;
-}
-  
-#history-button {
-    background: none;
-    border: none;
+.navbar {
+    list-style-type: none;
     padding: 0;
-    cursor: pointer;
+    margin: 0;
+    position: absolute;
+    top: 10px;
+    right: 10px;
+    border: 1px solid #ccc;
+    z-index: 1000;
 }
-  
-#history-button img {
+
+.menu-toggle {
+    display: inline-block;
+    cursor: pointer;
+    position: right;
     width: 40px;
-    height: 40px;
-    transition: filter 0.3s ease, transform 0.3s ease;
+    height: 30px;
+    margin: 10px;
 }
 
-nav > button:hover {
-    filter: brightness(0.8);
-    transform: scale(1.05);
+.bar {
+    display: block;
+    width: 100%;
+    height: 4px;
+    background-color: white;
+    margin: 6px 0;
+    transition: 0.4s;
 }
 
-#daily-button {
-    color:rgb(230, 230, 230);
-    border-style: solid;
-    border-color: rgb(230, 230, 230);
-    border-width: 4px;
-    background: none;
-    height: 40px;
-    width: 40px;
-    border-radius: 50%;
+.nav {
+    list-style-type: none;
     padding: 0;
-    margin: 2px;    
-}
\ No newline at end of file
+    margin: 0;
+    display: none;
+    position: absolute;
+    top: 50px;
+    right: 10px;
+    background-color: #333;
+    border: 1px solid #ccc;
+    z-index: 1000;
+}
+
+.nav-item {
```

Comment:
adding a hover effect would be good too there is feedback when user hovers ontop of the element